admin/installer/mysql/docker-compose-node1.yml

```diff
@@ -0,0 +1,10 @@
+database10:
```

Comment:
I think this file is unused? It be nice to either use it or at least mention it in some documentation so people know what it could be used for.